vitest.unit.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    pool: 'forks',
```

Comment:
Do we need it here? This is only needed when the tests fail to terminate for some mysterious reason. We have it in e2e but not in unit. Please check if it works without it

Reply:
My bad that I haven't actually tested/checked that attribute upon copy pasting from history api repo 🤦 

It's not needed, thanks for finding this